### PR TITLE
Run the binary size treemap script from the buildroot directory

### PR DESCRIPTION
**Pull request by jason-simmons**
This will produce a treemap where navigation starts from the buildroot and the flutter and third_party source directories are shown as siblings.

### CODE REVIEW

File: ci/builders/linux_android_aot_engine.json [hunk 1]
```diff
@@ -114,15 +114,11 @@
             "tests": [
                 {
                     "name": "Generate treemap for android_release_arm64",
-                    "language": "python3",
-                    "script": "third_party/dart/runtime/third_party/binary_size/src/run_binary_size_analysis.py",
+                    "language": "bash",
+                    "script": "flutter/ci/binary_size_treemap.sh",
                     "parameters": [
-                        "--library",
                         "../../src/out/android_release_arm64/libflutter.so",
```
Comment (zanderso) [inline ci/builders/linux_android_aot_engine.json:120]:
```suggestion
                        "../../src/out/android_release_arm64/libflutter.so"
```

File: ci/builders/linux_android_aot_engine.json [hunk 2]
```diff
@@ -114,15 +114,11 @@
...
-                        "--destdir",
                         "${FLUTTER_LOGS_DIR}",
```
Comment (zanderso) [inline ci/builders/linux_android_aot_engine.json:121]:
```suggestion
                        "${FLUTTER_LOGS_DIR}"
```

**Review by zanderso — APPROVED** [review]

File: ci/binary_size_treemap.sh [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+if [ "$(uname)" == "Darwin" ]; then
+  NDK_PLATFORM="darwin-x86_64"
+else
+  NDK_PLATFORM="linux-x86_64"
+fi
+ADDR2LINE="third_party/android_tools/ndk/toolchains/aarch64-linux-android-4.9/prebuilt/$NDK_PLATFORM/bin/aarch64-linux-android-addr2line"
+
+# Run the binary size script from the buildroot directory so that the treemap
+# path navigation will start from there.
+cd $ENGINE_BUILDROOT
```
Comment (zanderso) [inline ci/binary_size_treemap.sh:32]: Does this need to be "$ENGINE_BUILDROOT" to guard against spaces in the path? Same question about the usages of $INPUT_PATH and other variables below.

Comment (jason-simmons) [reply ci/binary_size_treemap.sh]: done

**Review by zanderso — APPROVED** [review]

**Pull request merged**